src/Numerics/LinearAlgebra/Matrix.Arithmetic.cs

```diff
@@ -1553,12 +1609,315 @@ public Matrix<T> PointwiseLog()
         /// <exception cref="ArgumentException">If this matrix and <paramref name="result"/> are not the same size.</exception>
         public void PointwiseLog(Matrix<T> result)
         {
-            if (ColumnCount != result.ColumnCount || RowCount != result.RowCount)
-            {
-                throw DimensionsDontMatch<ArgumentException>(this, result);
-            }
+            PointwiseUnary(DoPointwiseLog, result);
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        public Matrix<T> PointwiseAbs()
```

Comment:
Thanks. Yes, It's quite tedious; sorry for all the duplication (this would be less an issue with code gen, but we move away from that a long time ago for good reasons).